### ISSUE TITLE
any plan for ocr detection?

### ISSUE THREAD

**Issue by guanfuchen** [Topdu/OpenOCR#6]
Do you have plans for ocr detection task?

**Comment by Topdu** [comment]
Yes! Scene text detection and end-to-end methods are both on the schedule. We welcome any methods you are interested in for us to reproduce.

**Comment by cnszmike** [comment]
有train_rec.py，缺少train_det.py，希望用自己的数据集训练文本识别。
